src/classes/Contacts.cls

```diff
@@ -156,15 +192,50 @@ public inherited sharing class Contacts extends fflib_SObjects {
         Addresses.updateAddrIsOverride(isOverrideByAddressId);
     }
 
+    public void onBeforeUpdate_ACCT_IndividualAccounts_TDTM() {
+        Households.handleContactsBeforeUpdate(this);
+    }
+
     public void onAfterUpdate() {
         isUpdate = true;
         createAddressesForContacts();
     }
 
+    public void onAfterUpdate_ACCT_IndividualAccounts_TDTM(
+            TDTM_Runnable.DmlWrapper dmlWrapper) {
+        /**
+         * @description In the after trigger context, additional fields from the account
```

Comment:
Maybe it's just me, but having descriptions inside methods feels weird 🤷‍♂️ could make them block comments, or move them above the method name?

Reply:
Agreed.  Do you think this is a comment that is useful enough to keep? Or should it just be removed?  Not sure that it is worth keeping.

Reply:
I agree, just delete them (I see 3 areas that are like this)